docker-compose/run.sh

```diff
@@ -2,10 +2,18 @@
 
 set -eux
 
-worker="worker-ubuntu16-04"
+worker_volume="buildbarn-worker-ubuntu16-04"
+tini_volume="buildbarn-tini"
 
-sudo rm -rf bb "${worker}"
-mkdir -m 0777 "${worker}" "${worker}/build"
-mkdir -m 0700 "${worker}/cache"
+docker volume rm "${worker_volume}" || true
```

Comment:
Since you're moving to volumes, couldn't you simply `docker-compose down --volumes` instead?

Reply:
And same with creating them, why not simply `docker-compose up` ?

Reply:
> Since you're moving to volumes, couldn't you simply docker-compose down --volumes instead?

That would remove all of the volumes including the ones for the CAS, which would mean changing this project to effectively wipe the CAS between runs. (I don't see a way with `--volumes` to only delete specific volumes, [nor does the volumes top-level key seems to support only deleting a subset](https://docs.docker.com/compose/compose-file/#volume-configuration-reference).